src/Templates/src/templates/maui-lib/MauiLib1.csproj

```diff
@@ -0,0 +1,16 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+	<PropertyGroup>
+		<TargetFrameworks>net6.0;net6.0-ios;net6.0-android;net6.0-maccatalyst</TargetFrameworks>
+		<TargetFrameworks Condition="'$(OS)' == 'Windows_NT'">$(TargetFrameworks);net6.0-windows10.0.19041</TargetFrameworks>
```

Comment:
Not sure what the plan is, but is this version of win10 correct? Or is it too high?

Reply:
That is the TFM, but the minimum version set later will make it work on older OS.